tests/sockshop/expected_netpol_output.json

```diff
@@ -309,6 +330,21 @@
                         "name": "rabbitmq"
                     }
                 },
+                "ingress": [
+                    {
+                        "ports": [
+                            {
+                                "protocol": "TCP",
```

Comment:
btw, noticed that `rabbitmq` svc has `prometheus.io/scrape: 'true'`, but the deployment has `prometheus.io/scrape: "false"`

Reply:
yes, I'm not quite sure what was the user intent in this case, neither how Prometheus will behave.
Doesn't seem to harmful to open the specified port anyway.